.github/workflows/extension-test-on-pr.yml

```diff
@@ -263,23 +263,29 @@ jobs:
         id: run_tests
         working-directory: meshery-extensions
         run: |
-          echo 'test-results<<EOF' >> $GITHUB_OUTPUT
-          make test >> $GITHUB_OUTPUT
-          echo 'EOF' >> $GITHUB_OUTPUT
+          make test
         env:
           PROVIDER_TOKEN: ${{ secrets.PROVIDER_TOKEN }}
           MESHERY_SERVER_URL: "http://localhost:9081"
           REMOTE_PROVIDER_URL: "https://meshery.layer5.io"
           REMOTE_PROVIDER_USER_EMAIL: ${{ secrets.LAYER5_CLOUD_TESTING_BOT_EMAIL }}
           REMOTE_PROVIDER_USER_PASSWORD: ${{ secrets.LAYER5_CLOUD_TESTING_BOT_PASSWORD }}
-
+      - name: Test reporter
+        uses: actions/github-script@v7
+        id: test-reports
+        with:
+          script: |
+              const fs = require('fs')
+              const E2ETestReporter = String(fs.readFileSync('./meshmap/test-report.md'))
```

Comment:
why are we not adding the custom reporter in playwright config

Reply:
@aabidsofi19 The custom report has already been added to the Meshery extension repo, this file is the result of the reporting which is separated from logs